data_diff/dbt_parser.py

```diff
@@ -140,7 +162,7 @@ def get_connection_creds(self) -> Tuple[Dict[str, str], str]:
 
         return credentials, conn_type
 
-    def set_connection(self):
+    def set_connection(self) -> dict:
```

Comment:
This method has no single `return` in it, so it returns `None`, doesn't it?

Reply:
ty for catching this, fixed